btcjson/walletsvrcmds.go

```diff
@@ -640,6 +641,19 @@ func NewWalletPassphraseChangeCmd(oldPassphrase, newPassphrase string) *WalletPa
 	}
 }
 
+// VersionCmd defines the version JSON-RPC command.
+//
+// NOTE: This is a btcsuite extension ported from
+// github.com/decred/dcrd/dcrjson.
+type VersionCmd struct{}
```

Comment:
This is a btcd-specific extension command, so I think it should be in the `btcdextcmds.go` file.  This file is for wallet-specific commands that are compatible with Core.